app/src/main/java/com/tvtracker/ui/main/BrowseViewModel.kt

```diff
@@ -18,11 +18,11 @@ import kotlinx.coroutines.launch
 
 class BrowseViewModel(var mediaService: IMediaService = MediaService()): ViewModel() {
 
-    private lateinit var firestore: FirebaseFirestore
+    private var firestore: FirebaseFirestore = FirebaseFirestore.getInstance()
     var user: User? = null
     var selectedMediaItem by mutableStateOf(MediaItem())
 
-    val PAGE_SIZE = 10
```

Comment:
Renamed Page size value to fit with best practices

Reply:
If it's a constant, which appears to be the case here, PAGE_SIZE is the correct way to name it.